myuw/data/resource_link_import.csv

```diff
@@ -54,17 +54,17 @@ Services for Faculty and Staff,Research,all,https://sage.admin.uw.edu/era/uwneti
-Services for Faculty and Staff,Procurement,all,,,http://f2.washington.edu/fm/ps,Procurement Services,https://www.uwb.edu/finance/purchasing,Purchasing,http://www.tacoma.uw.edu/node/40039,Procurement Services,no
+Services for Faculty and Staff,Procurement,all,https://finance.uw.edu/ps/how-to-buy/ariba,ARIBA,,,,,,,no
+Services for Faculty and Staff,Procurement,all,,,https://finance.uw.edu/ps/,Procurement Services,https://www.uwb.edu/finance/purchasing,Purchasing,http://www.tacoma.uw.edu/node/40039,Procurement Services,no
 Services for Faculty and Staff,Others,all,http://registrar.washington.edu/staffandfaculty/electronic-academic-records-system-ears,Electronic Advising Records System (EARS),,,,,,,no
-Services for Faculty and Staff,Others,all,,,http://washington.edu/itconnect,IT Connect,https://www.uwb.edu/it,Information Technologies,http://www.tacoma.uw.edu/information-technology/information-technology,Information Technology,no
+Services for Faculty and Staff,Others,all,,,https://itconnect.uw.edu/,IT Connect,https://www.uwb.edu/it,Information Technologies,http://www.tacoma.uw.edu/information-technology/information-technology,Information Technology,no
 Services for Faculty and Staff,Others,all,https://itconnect.uw.edu/wares/,Software and Hardware,,,,,,,
 Services for Faculty and Staff,Others,all,,,https://www.ehs.washington.edu/,Environmental Health & Safety,https://www.uwb.edu/facility/ehs,Environmental Health & Safety,http://www.tacoma.uw.edu/node/38910,Environmental Health & Safety,no
 Services for Faculty and Staff,Others,all,,,https://facilities.uw.edu/,UW Facilities,https://www.uwb.edu/facility,Facilities and Campus Operations,http://www.tacoma.uw.edu/facilities-services/facilities-services,Facilities Services,no
 Services for Faculty and Staff,Others,all,,,https://facilities.uw.edu/services/requests,UW Facilities: Request Service,http://dlweb.megamation.com/uwb/DLWEB.php/O4W_INFO_PAGE,Facilities Maintenance Request,http://www.tacoma.uw.edu/facilities-services/work-order-request,Facilities Services: Work Order Request,no
 Campus Life,Campus Retail,all,,,http://www.bookstore.washington.edu/student_faculty/student_faculty.taf?verba=uwseattle,University Book Store,http://www.bookstore.washington.edu/student_faculty/student_faculty.taf?verba=uwbothell,University Book Store,http://www.bookstore.washington.edu/student_faculty/student_faculty.taf?verba=uwtacoma,University Book Store,no
 Campus Life,Campus Safety,all,,,,,,,http://www.tacoma.uw.edu/campus-safety/,Campus Safety & Security,no
-Campus Life,Campus Safety,all,,,http://www.washington.edu/safecampus/,SafeCampus,http://www.washington.edu/safecampus/uwb/,SafeCampus,http://www.washington.edu/safecampus/uwt/,SafeCampus,no
+Campus Life,Campus Safety,all,,,http://www.washington.edu/safecampus/,SafeCampus,https://www.uwb.edu/safety,SafeCampus,https://www.tacoma.uw.edu/node/38890,SafeCampus,no
```

Comment:
I think we should remove the specific links for UWB and UWT here and instead just make SafeCampus viewable for all campuses. 

The Campus Safety Department (https://www.uwb.edu/safety) can be moved to it's own line/entry, or be included in line 66, which shows UWT Campus Safety & Security